.tool-versions

```diff
@@ -0,0 +1 @@
+pnpm 8.5.1
```

Comment:
Can we unlock the version?

Reply:
Good catch, shold not have this in the PR. Thanks.